index.html

```diff
@@ -73,6 +73,238 @@ <h4>Glad to see you!</h4>
+            </div>
+            <div class="project-action-btn">
+              <button class="detail-btn">See project</button>
+            </div>
+          </div>
+        </div>
+        <div class="project">
+          <div class="grid-item">
+            <img
+              class="portfolio-snapshot"
+              src="./images/snapshot-3.png"
+              alt="portfolio-snapshot"
+            />
+          </div>
+          <div class="left-block">
+            <div class="project-details">
+              <h2 class="project-title">Tonic</h2>
+              <ul class="project-stack">
+                <li><h3 class="project-sub-title">CANOPY</h3></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">Back End Dev</h4></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">2015</h4></li>
+              </ul>
+            </div>
+            <div>
+              <p class="project-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="project-action-btn">
+              <button class="detail-btn">See project</button>
+            </div>
+          </div>
+        </div>
+        <div class="project">
+          <div class="grid-item">
+            <img
+              class="portfolio-snapshot"
+              src="./images/snapshot-4.png"
+              alt="portfolio-snapshot"
+            />
+          </div>
+          <div class="left-block">
+            <div class="project-details">
+              <h2 class="project-title">Multi-Post Stories</h2>
+              <ul class="project-stack">
+                <li><h3 class="project-sub-title">CANOPY</h3></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">Back End Dev</h4></li>
+                <li><img src="./images/counter.png" alt="counter" /></li>
+                <li><h4 class="project-year-role">2015</h4></li>
+              </ul>
+            </div>
+            <div>
+              <p class="project-desc">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div>
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+            </div>
+            <div class="project-action-btn">
+              <button class="detail-btn">See project</button>
```

Comment:
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed` and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/228013110-a6708530-0861-41ee-8767-f2f043d75810.gif)

- Figma design
<img width="446" alt="right" src="https://user-images.githubusercontent.com/41163571/227707940-ae12a5de-15d8-4b79-b06b-a16bfdafd017.png">

- For the two types of buttons I went through.